_episodes/03-packages.md

```diff
@@ -101,39 +100,42 @@ of software (if operating just in the shell), is `apt policy`.
   `PACKAGE`.
 
 
-Indispensible resource for reproducibility is the 
+An indispensable resource for reproducibility is the 
 [Debian snapshots](http://snapshots.debian.org) archive. It provides access to 
-the states of all aforementioned APT repositories, as they were provided in the past.
-This allows to obtain any previously present in those APT repositories version of
+the states of all aforementioned APT repositories as they were provided in the past.
+This resource allows us to obtain anything previously present in those APT repositories’ versions of
 the package.
 
 
 ## NeuroDebian
 
-[NeuroDebian](http://neuro.debian.net) project was established to integrate
+The [NeuroDebian](http://neuro.debian.net) project was established to integrate
 software used for research in psychology and
-neuroimaging within standard Debian distribution.  The majority of the packages 
+neuroimaging within the standard Debian distribution. The majority of the packages 
 maintained by the NeuroDebian team get uploaded to Debian unstable and then 
 propagate to Debian and Ubuntu releases.
 
 To facilitate access to the most recent versions
-of such software on already existing releases of Debian and its most popular 
+of such software on pre-existing releases of Debian and its most popular 
 [derivative](https://wiki.debian.org/Derivatives) [Ubuntu](http://ubuntu.com),
-NeuroDebian project established its own 
-[APT](https://en.wikipedia.org/wiki/APT_(Debian)) repository.  So, in a vein, 
-such repository is similar to [Debian backports](https://backports.debian.org/)
-repository, but a) it also 
-supports Ubuntu releases, b) typically backport builds are 
-uploaded to NeuroDebian as soon as they are uploaded to Debian unstable, c) contains
-some packages which did not make it to Debian proper yet.
-
-To enable NeuroDebian on your standard Debian or Ubuntu machine, you could 
+the NeuroDebian project established its own 
+[APT](https://en.wikipedia.org/wiki/APT_(Debian)) repository. So, in a vein, 
+this repository is similar to the [Debian backports](https://backports.debian.org/)
+repository, but has several differences:
+
+a) it also supports Ubuntu releases, and
+
+b) typically backport builds are uploaded to NeuroDebian as soon as they are uploaded to Debian unstable, and
+
+c) it contains some packages that have not yet made it to Debian proper.
+
+To enable NeuroDebian on your standard Debian or Ubuntu machine, please 
 `apt-get install neurodebian` (and follow the interactive dialogue) or just follow 
-the instructions on http://neuro.debian.net .
+the instructions on http://neuro.debian.net
 
 If you are using [Docker](http://docker.io), 
 [NeuroDebian Docker images](https://hub.docker.com/_/neurodebian/) are provided for all 
-supported Debian and Ubuntu releases.  If you are using [Singularity]()
+supported Debian and Ubuntu releases. If you are using [Singularity], then please standby for instructions.
```

Comment:
This sentence was left incomplete in the original version so please advise on its content! @yarikoptic